db/workflow.go

```diff
@@ -345,7 +345,7 @@ func GetWorkflowDataVersion(ctx context.Context, db *sql.DB, workflowID string)
 }
 
 // GetfromWfWorkflowTable : gives you the current workflow
-func GetfromWfWorkflowTable(ctx context.Context, db *sql.DB, id string) ([]string, error) {
+func GetfromWfWorkflowTable(db *sql.DB, id string) ([]string, error) {
```

Comment:
I am not sure this function name describes what it does. I understand it was already like that but can we evaluate the idea to change it to something different? Like `SelectWorkflowByID`?

Kind of like the idea to use SQL like keywords to describe functions that interact with SQL database 🍬

Reply:
This function does not used by any external user. I will update the comment though.

Reply:
Even internal users need clarity :)